CONTRIBUTING.md

```diff
@@ -53,3 +53,92 @@ To build a local image use the `bin/task docker-local` command in the [FerretDB]
 To use a pre-built image you must set the `FERRETDB_IMAGE` environment variable, e.g. `export FERRETDB_IMAGE=ghcr.io/ferretdb/ferretdb-dev:main`.
 
 As mentioned above, this approach is not recommended.
+
+## How to write diff tests
+
+Diff tests demonstrate how FerretDB differs from MongoDB.
+
+We write diff tests when we can't write "regular" integration or compat tests because FerretDB behaves differently
+from MongoDB in some cases.
+If such differences are visible to users, in addition to diff tests, they must be documented
```

Comment:
If they are not visible to users, they should not be diff tests

Reply:
@AlekSi, this is not quite true. We have a diff test for `debugError` command, which we don't mention in the users' diff documentation.

Reply:
Ok, let's mark it as an exception then. I don't think there will be others

Reply:
fixed